test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/Feeds/MultiSourcePackageMetadataProviderTests.cs

```diff
@@ -69,6 +69,35 @@ public async Task GetLocalPackageMetadataAsync_WhenGlobalSourceHasPackage_Withou
                 Assert.Null(await metadata.GetDeprecationMetadataAsync());
             }
 
+            [Fact]
+            public async Task GetLocalPackageMetadataAsync_WhenGlobalSourceHasPackage_WithoutVulnerabilityMetadata()
```

Comment:
Would expect a test for the positive case, to ensure various vulnerability data *does* show up when expected. Consider 2+ sources with varying metadata, and ensure the worst case is found.